x-pack/plugins/watcher/server/routes/api/watch/register_save_route.ts

```diff
@@ -5,98 +5,104 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import { RequestHandler } from 'kibana/server';
+import { IScopedClusterClient } from 'kibana/server';
 import { i18n } from '@kbn/i18n';
 import { WATCH_TYPES } from '../../../../common/constants';
 import { serializeJsonWatch, serializeThresholdWatch } from '../../../../common/lib/serialization';
-import { callWithRequestFactory } from '../../../lib/call_with_request_factory';
 import { isEsError } from '../../../lib/is_es_error';
 import { RouteDependencies } from '../../../types';
 import { licensePreRoutingFactory } from '../../../lib/license_pre_routing_factory';
 
-function fetchWatch(callWithRequest: any, watchId: string) {
-  return callWithRequest('watcher.getWatch', {
+const paramsSchema = schema.object({
+  id: schema.string(),
+});
+
+const bodySchema = schema.object(
+  {
+    type: schema.string(),
+    isNew: schema.boolean(),
```

Comment:
@jloleysens Is it possible that this is a breaking change? Starting in 7.7 APM has received this bug report:
```
[request body.isNew]: expected value of type [boolean] but got [undefined]
```

It seems `isNew` was changed from optional to required.

Reply:
We will fix this in 7.8 by supplying it but other consumers might run into the same issue.